force-app/main/sms/classes/HOT_ReminderSMSService.cls

```diff
@@ -162,7 +144,9 @@ public without sharing class HOT_ReminderSMSService {
             sms.Thread__c = thread.Id;
             sms.Recipient__c = (thread.CRM_Thread_Type__c == 'HOT_BESTILLER-FORMIDLER' && ordererPhoneOnRequest.get(thread.CRM_Related_Object__c) != null) ? ordererPhoneOnRequest.get(thread.CRM_Related_Object__c) : thread.CRM_Account__r.CRM_Person__r.INT_KrrMobilePhone__c;
```

Comment:
Heller utvide getRecipient()?

Reply:
getRecipient tar WO's. Kanskje like greit å gjøre sjekken her enn å ha en egen metode for Thread__c?

Reply:
getRecipient tar WOs. Oppretter ny getRecipient for Thread__c 👍🏻